frontend/src/pages/modelRegistry/screens/ModelRegistrySelector.tsx

```diff
@@ -32,12 +32,14 @@ type ModelRegistrySelectorProps = {
   modelRegistry: string;
   onSelection: (modelRegistry: string) => void;
   primary?: boolean;
+  isCatalogModel?: boolean;
 };
 
 const ModelRegistrySelector: React.FC<ModelRegistrySelectorProps> = ({
   modelRegistry,
   onSelection,
   primary,
+  isCatalogModel,
```

Comment:
I think this prop can just be called `isFullWidth`, there may be other forms in the future other than catalog where we want to use the ModelRegistrySelector.